data/tlg0081/tlg007/tlg0081.tlg007.1st1K-grc1.xml

```diff
@@ -0,0 +1,4807 @@
+littera erasa M τῆς del Krüger 18 εὐουλίδην BP 20 εἴ-
+κοσι MBPv 22 ὁποῖα λέγω] οἷς οἶδα ἐγὼ M ΒPv Ammaeum
+adloquitur cf p157,16 διμοσίων P 23 ἔφερον ex ἔφερων M
+ἔφερεν in mg ἔφερον B</note>
+
+<pb n="157"/>
+ἔμελλε τοῦ μετρίου γενήσεσθαι ὁ λόγος. ἐν οἷς δῆλός
+ἐστι περὶ καλλιλογίαν καὶ σεμνότητα καὶ πάσας τὰς
+ἐπιθέτους κατασκευὰς <add cause="omitted">οὐ</add> μᾶλλον ἐσπουδακὼς ἢ περὶ
+<milestone unit="994"/> τὴν ἀκρίβειαν. ὁ δὲ πρὸς τὴν ἐπιστολὴν καὶ τοὺς
+πρέσβεις τοὺς παρὰ Φιλίππου ῥηθεὶς λόγος, ὃν ἐπιγράφει <lb n="5"/>
+Καλλίμαχος ὑπὲρ Ἁλοννήσου, ὁ τὴν ἀρχὴν
+τήνδε ἔχων· Ὦ ἄνδρες Ἀθηναῖοι, οὐκ ἔστιν, ὅπως αἱ
+αἰτίαι, ἂς Φίλιππος αἰτιᾶταιʼ ὅλος ἐστὶν ἀκριβὴς καὶ
+λεπτὸς καὶ τὸν Λυσιακὸν χαρακτῆρα ἐκμέμακται εἰς
+ὄνυχα, ἐξαλλαγῆς δὲ ἢ σεμνολογίας ἢ δεινότητος ἢ <lb n="10"/>
+τῶν ἄλλων τινός, ἃ τῇ Δημοσθένους δυνάμει παρακολουθεῖν
+πέφυκεν, ὀλίγην ἐπίδειξιν ἔχει. τίς οὖν
+ἐστι κἀν τούτοις ἡ διαφορά; καὶ πῶς ἂν διαγνοίη τις,
+ὅταν εἰς τὸν ἀναγκαῖον καταβῇ χαρακτῆρα ὁ Δημοσθένης,
+πῇ κρείττων ἐστὶ Λυσίου καὶ κατὰ τὴν λέξιν; <lb n="15"/>
+ἀξιοῖς γὰρ δὴ καὶ τοῦτο μαθεῖν. φυσική τις ἐπιτρέχει
+τοῖς Λυσίου λόγοις εὐστομία καὶ χάρις, ὥσπερ ἔφην
+καὶ πρότερον, ἧ προὔχει πλὴν Δημοσθένους τῶν ἄλλων
+ῥητόρων. αὕτη μέντοι, καθάπερ νότιός τις αὖρα, μέχρι
+<milestone unit="995"/> προοιμίου καὶ διηγήσεως αὐτὸν ἄγει, ὅταν δʼ εἰς τοὺς <lb n="20"/>
+ἀποδεικτικούς ἔλθῃ λόγους, ἀμυδρά τις γίνεται καὶ
+ἀσθενής, ἐν δὲ δὴ τοῖς παθητικοῖς εἰς τέλος ἀποσβέννυται·
+τόνος γὰρ οὐ πολύς αὐτῇ πρόσεστιν οὐδʼ ἰσχύς.
+<note type="footnote">7 [Demosthenes] περὶ Ἁλοννήσου or VII 1</note>
+<note type="footnote">1 γενήσεσθαι B mg, M sed γενή postea additum: γενῆσθαι
+P, B in textu, γενέσθαι v 2 καλλολογίαν M B 3 οὐ add Vliet</note>
+<note type="footnote">4 πρὸς] περὶ MBPv 5 λόγον mg B 6 immo περὶ | ἀλο-
+νήσου MBPv 7 ἔχων om V 10 ἐξαλαγῆς BP 13 ἡ διμοσθέ-
+νους δυνάμει παρα ἡ διαφορὰ B1 διαγνοίην P 14 ἀναγκαῖον]
+Λυσιακὸν malit Usener 15 πὴ M, B mg: τῆ BP 20 δʼ M, B mg:
+om BP 21 ἔλθει P</note>
+
+<pb n="158"/>
+παρὰ δὲ τῷ Δημοσθένει πολὺς μὲν ὁ τόνος, αὐτάρκης
+δʼ ἡ χάρις, ὥστε κἀν ταύτῃ τῷ διαρκεῖ καὶ μετρίῳ
+νικᾶν κἀν ἐκείνῳ τῷ παντὶ προέχειν. τοῦτο παρατήρημα
+δεύτερον, ᾧ διαγνοίη τις ἂν τὴν Δημοσθένους
+<lb n="5"/> διάλεκτον, ὅταν εἰς ταῦτα τἀναγκαῖα συνάγηται. οὐ
+γὰρ ὥσπερ τὴν ἐξαλλαγὴν καὶ περιττολογίαν καὶ πάντας
+τοὺς ἐπιθέτους ἐκδύεται κόσμους, οὕτως καὶ τὸ
+μέγεθος καὶ τὸν τόνον, ἀλλʼ ἔστιν αὐτῆς ἀναφαίρετος
+οὗτος εἴτʼ ἄρα συγγενὴς εἴτε ἀσκήσει παρὼν εἰς ῥῆσιν.
+<lb n="10"/> ἐπιτάσεις μέντοι καὶ ἀνέσεις λαμβάνει τινὰς ἀναλόγους.
+καὶ ταῦτʼ ἤδη γνώριμα οἷς λέγω, καὶ οὐθὲν δεόμεθα
+παραδειγμάτων.</p>
+</div>
+
+<div type="textpart" subtype="chapter" n="14">
+<p>14 ὥστε περὶ μὲν τοῦ χαρακτῆρος . τοῦ δὲ
+ῥητορικοῦ γένους τοῦ μεταξύ τῶν ἄκρων ἑκατέρων, ὃ
+<lb n="15"/> ἀτελὲς παραλαβὼν ὁ Δημοσθένης παρʼ Ἰσοκράτους τε
+καὶ ἔτι προτέρου Θρασυμάχου καὶ τελευταίου Πλάτωνος <milestone unit="996"/>
+ἐτελείωσεν ὅσον ἦν ἀνθρωπίνη φύσει δυνατόν,
+πολλὰ μ
```

Comment:
There are section-divs in this chapter (and only in this one) but the CTS only has one level. Shouldn't the sections appear there too?